kitsune/sumo/tests/test_googleanalytics.py

```diff
@@ -32,7 +32,7 @@ def test_visitors_by_locale(self, _build_request):
         visits = googleanalytics.visitors_by_locale(
             date(2013, 01, 16), date(2013, 01, 16))
 
-        eq_(58, len(visits))
+        eq_(59, len(visits))
```

Comment:
Do you have more info why did this changed?

Reply:
`Malay` locale was added in kitsune with #2888.
Therefore, there are total 59 locale.

Reply:
Hmmm OK but the test is limited to Jan 2013 which is before this change. Also this test doesn't seem to fail anymore

Reply:
I updated the test in #2965 with this change. Thanks!